Kopernicus/Kopernicus/Configuration/PropertiesLoader.cs

```diff
@@ -209,7 +209,7 @@ public NumericParser<double> navballSwitchRadiusMultLow
 
             // Biomes of this body
             [PreApply]
-            [ParserTargetCollection("Biomes", nameSignificance = NameSignificance.None)]
+            [ParserTargetCollection("Biomes", nameSignificance = NameSignificance.Key, key = "Biome")]
```

Comment:
Please revert that. It isn't actually needed, but it will break those mods who use different names